[doc] add monitoring session

panpan0000 opened the pull request:
refer to original example/prometheus docs and https://github.com/LMCache/LMCache/pull/1318 


add how to monitor LMCache session.

panpan0000 committed:
[doc] add monitoring session

Signed-off-by: Peter Pan <Peter.Pan@daocloud.io>

panpan0000 commented:
anyone ?

sammshen reviewed (approved):
LGTM! @panpan0000 would it be possible to add an example grafana dashboard if you have one on your end?

panpan0000 commented:
> LGTM! @panpan0000 would it be possible to add an example grafana dashboard if you have one on your end?

not yeah..  @sammshen  but good idea. 

 @fkajerry  can you do it ?

fkajerry commented:
> > LGTM! @panpan0000 would it be possible to add an example grafana dashboard if you have one on your end?
> 
> not yeah.. @sammshen but good idea.
> 
> @fkajerry can you do it ?

yes, I am very delighted to.

panpan0000 commented:
still need a +1 ,anyone ?

sammshen commented:
@maobaolong would you like to take a look? this has some internal api server documentation!

ApostaC reviewed (approved):
LGTM!

There are more Prometheus metrics, but we can gradually update the doc.

maobaolong commented:
> @maobaolong would you like to take a look? this has some internal api server documentation!
Thanks for your remind, but sorry for the late reply.

@panpan0000 Thanks for this contribution, I noticed that you introduce a threshold of `internal_api_sever`, but in fact I have submitted and merged a document of `internal_api_server`, it placed here https://docs.lmcache.ai/internal_api_server/internal_api_server.html . You can feel free to move it to the new place to follow your new layout, in fact, I like your layout.

panpan0000 committed:
[docs] adjust internal api server and metrics

Signed-off-by: Peter Pan <Peter.Pan@daocloud.io>

panpan0000 commented:
Thank you both

1.  @maobaolong  I adjusted the session in observability part( I'm on purpose to provide an example of env variable for internal-api-server ,as a replenish for https://docs.lmcache.ai/internal_api_server/internal_api_server.html )  , and provide a link to that doc at the end.

<img width="893" height="948" alt="image" src="https://github.com/user-attachments/assets/0ffac81c-5534-4f18-9251-d0ba226749d5" />






---

2. also , I updated metrics per @ApostaC 

<img width="622" height="885" alt="image" src="https://github.com/user-attachments/assets/745d52e4-d11a-4098-884c-c54b4451b7d3" />

maobaolong reviewed (approved):
LGTM
